### PR TITLE
[CCI] Update tabbable dependency

### PR DESCRIPTION
**Pull request by andreymyssak**
### Description
Update tabbable dependency to 6.1.1
 
### Issues Resolved
https://github.com/opensearch-project/oui/issues/594
 
### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [X] All tests pass
  - [X] `yarn lint`
  - [X] `yarn test-unit`
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/oui/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: i18ntokens.json
Comment (BSFishy) [inline i18ntokens.json:1]: Why are these being changed in this PR? I don't even see some of these files being edited

File: src/components/context_menu/context_menu_panel.tsx [hunk 1]
```diff
@@ -270,7 +270,10 @@ export class OuiContextMenuPanel extends Component<Props, State> {
 
         // Otherwise let's focus the first tabbable item and expedite input from the user.
         if (this.content) {
-          const tabbableItems = tabbable(this.content);
+          const tabbableItems = tabbable(this.content, {
+            displayCheck: 'legacy-full',
+          });
```
Comment (BSFishy) [inline src/components/context_menu/context_menu_panel.tsx:275]: Why is this the only instance where options are added to the `tabbable` call? Is it special for some reason? Should the other calls to `tabbable` have it too?

File: i18ntokens.json
Comment (andreymyssak) [reply i18ntokens.json:1]: I left a comment about this in another PR https://github.com/opensearch-project/oui/pull/609#discussion_r1154684969. It seems that any change in the code affects changes to the `i18tokens.json` file.

File: src/components/context_menu/context_menu_panel.tsx (hunk 1 above)
Comment (andreymyssak) [reply src/components/context_menu/context_menu_panel.tsx:275]: The rest of the cases do not cause problems with the tests and smoothly support the transition to the new version. In this file apparently a different case, after the upgrade the tests did not behave correctly, so I had to apply `legacy-full`.

File: yarn.lock [hunk 2]
```diff
@@ -13543,7 +13538,7 @@ request-promise-native@^1.0.5:
     stealthy-require "^1.1.1"
     tough-cookie "^2.3.3"
 
-request@^2.87.0, request@^2.88.0:
```
Comment (BSFishy) [inline yarn.lock:13546]: This is changed in #652, need to keep an eye on it so we don't have merge conflicts

**Review by BSFishy — APPROVED** [review]

**Review by ashwin-pc — APPROVED** [review]
Thanks for the change @andreymyssak! 🎉

File: i18ntokens.json
Comment (joshuarrrr) [reply i18ntokens.json:1]: @andreymyssak Do you mind opening a new research issue to look into this? It seems like something we may want to tweak or optimize.

File: src/components/context_menu/context_menu_panel.tsx (hunk 1 above)
Comment (joshuarrrr) [reply src/components/context_menu/context_menu_panel.tsx:275]: Let's open a follow-up issue to see if we can refactor and remove.

**Review by joshuarrrr — APPROVED** [review]

**Pull request merged**